prow/plank/plank_test.go

```diff
@@ -690,6 +690,11 @@ func TestBatch(t *testing.T) {
 	if fc.prowjobs[0].Status.State != kube.FailureState {
 		t.Fatalf("Wrong state: %v", fc.prowjobs[0].Status.State)
 	}
+
+	// This is what the SQ reads.
+	if fc.prowjobs[0].Spec.Context != "Some Job Context" {
+		t.Fatalf("Wrong context: %v", fc.prowjobs[0].Spec.Context)
```

Comment:
Because this isn't a table-driven test there's no question of "which testcase broke", so it's kindof a grey area.

Mostly I was just lazy, though.